src/menu/menu.scss

```diff
@@ -149,6 +150,32 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
       }
     }
   }
+
+  &-skeleton {
+    --iui-menu-item-skeleton-max-width: 30%;
+    cursor: auto;
+
+    .iui-content {
+      max-width: var(--iui-menu-item-skeleton-max-width);
```

Comment:
I wasn't sure about this variable but it allows easier to override if users decide to make some changes trough css classes.